public/docs/ts/latest/guide/router.jade

```diff
@@ -122,7 +126,7 @@ include ../_util-fns
 
 :marked
```

Comment:
Should all references to `router` be changed to `featuremodulenameRouterModule` ? I think we can skip this for now and schedule this for a future PR. Thoughts?